dbt_platform_helper/templates/pipelines/codebase/overrides/stack.ts

```diff
@@ -115,7 +116,7 @@ export class TransformedStack extends cdk.Stack {
                 location: `https://github.com/${this.codebaseConfiguration.repository}.git`,
                 gitCloneDepth: 0,
                 auth: {type: 'OAUTH'},
-                gitSubmodulesConfig: {fetchSubmodules: true},
+                gitSubmodulesConfig: {fetchSubmodules: false},
```

Comment:
When this is set to true, the build will fail at the `DOWNLOAD_SRC` phase before it reaches the pre-build script and we can edit the submodules file. Is there a scenario where we would need this to be true?

Reply:
I don't think so, unless there was something in an install phase that used a public repo with submodules.